src/main/java/com/ibm/eventstreams/kafkaconnect/plugins/xml/engines/StructToXmlBytes.java

```diff
@@ -331,14 +320,11 @@ private void addArrayElements(Document doc, Element parentElement, Struct source
             final Element element = doc.createElement(field);
             if (obj instanceof Struct) {
                 processStruct(doc, element, (Struct) obj);
-            }
-            else if (obj instanceof Collection) {
+            } else if (obj instanceof Collection) {
                 addListElements(doc, element, (Collection<?>) obj, "entry");
-            }
-            else if (obj instanceof Map) {
+            } else if (obj instanceof Map) {
```

Comment:
```suggestion
            } 
            else if (obj instanceof Map) {
```

Unrelated to this specific PR